ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java

```diff
@@ -649,16 +652,47 @@ public synchronized void measure(int tag, int[] outputBuffer) {
     if (rootView == null) {
       throw new NoSuchNativeViewException("Native view " + tag + " is no longer on screen");
     }
-    rootView.getLocationInWindow(outputBuffer);
+    computeBoundingBox(rootView, outputBuffer);
```

Comment:
Here we could still use `rootView.getLocationInWindow(outputBuffer)`, but I assume you used `computeBoundingBox` for consistency? We don't need the width and the height (only x and y) of the root view and it's not supposed to be transformed, so `getLocationInWindow` should still be sufficient.

Reply:
Yes consistency and cancellation of the missing offsets in my version of mapRectFromViewToScreenCoords. Have you read my annotations on the original code in response to mdvacca? By using computeBoundingBox twice, offsets cancel each other and I don't need to port this part of the function (hard to do since everything is private).

```js
    // I think the 2 following offsets would be the same for the measured view and 
    // the React root view. Since we subtract the root view position from the 
    // measured view position in measure(), these offsets are cancelled.
    // In short: (a + offsets) - (b + offsets) = a - b
    // 
    // class ViewRootImpl is hidden, I could not find a way to rewrite this code.
    // Since we don't need it for our purpose, I removed it.
    if (parent instanceof ViewRootImpl) {
        ViewRootImpl viewRootImpl = (ViewRootImpl) parent;
        rect.offset(0, -viewRootImpl.mCurScrollY);
    }
    rect.offset(mAttachInfo.mWindowLeft, mAttachInfo.mWindowTop);
```

Reply:
Thank you for the explanation. I think I got the idea now: if we call `rootView.getLocationInWindow(outputBuffer)` instead of `computeBoundingBox(rootView, outputBuffer)`, `getLocationInWindow` internally will calculate the offset for the rootView:

```
if (parent instanceof ViewRootImpl) {
    ViewRootImpl viewRootImpl = (ViewRootImpl) parent;
    rect.offset(0, -viewRootImpl.mCurScrollY);
}
```

And if mCurScrollY != 0 for our root view, the bounding box rect of the child view will be missing this offset since we don't have access to the private API of ViewRootImpl to calculate it accordingly.